### PR TITLE
Update HurumapChart

### PR DESCRIPTION
**Pull request by karimkawambwa**
## Description

Update rendering hurumap chart in analysis

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Screenshots

localhost_3000_profiles_kenya_development-context-5

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/components/AnalysisContent/index.js [hunk 1]
```diff
@@ -98,26 +97,16 @@ function AnalysisContent({
         return indicator;
       });
     }
-
-    const hurumapCharts = Array.from(
-      document.querySelectorAll('div[data-chart-id]')
+    setHurumapCharts(
+      Array.from(document.querySelectorAll('div[data-chart-id]')).map(
```
Comment (kilemensi) [inline src/components/AnalysisContent/index.js]: QQ: Shouldn't `data-chart-id` have been `data-hurumap-chart-id` or something? I'm just thinking when we have HURUmap and Flourish on the page and we need to _hydrate_ both of them.

File: src/components/AnalysisContent/index.js [hunk 2]
```diff
@@ -98,26 +96,16 @@ function AnalysisContent({
...
-              <HURUmapChart geoId={geoId} chartId={chartId} charts={charts} />
-            </ThemeContainer>
-          </ApolloContainer>,
-          chart
-        );
-        return chart;
-      });
-    }
-  }, [charts, takwimu.country.name, topicIndex]);
+  }, [charts, takwimu.country.name]);
```
Comment (KhadijaMahanga) [inline src/components/AnalysisContent/index.js]: @karimkawambwa we need `topicIndex` on this useEffect. Hurumap charts are only rendered on the initial visit & initial topic. And not rendered once you navigate from one topic to the next.

**Comment by KhadijaMahanga** [comment]
also, mbona hiii imebadilika? `Other topics in..` replaced by topic title and `Read Next` title haipo Screenshot from 2019-10-29 10-54-24

Could not be associated with this PR but if you look into that

**Comment by karimkawambwa** [comment]
> also, mbona hiii imebadilika? `Other topics in..` replaced by profile title and `Read Next` title haipo
> Could not be associated with this PR but if you look into that

@KhadijaMahanga these are the things that need to go into wordpress. 
Wagtail provided these titles just because they have potential to change languages i believe that was the idea.

Definitely something for another PR.

**Pull request merged**